bin/visualize_packs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "pathname"
+require "optparse"
+require "ostruct"
+
+require_relative '../lib/visualize_packs'
+require_relative '../lib/options'
+
+options = Options.new
+
+OptionParser.new do |opt|
+  opt.on('--no-layers', "Don't show architectural layers") { |o| options.show_layers = false }
+  opt.on('--no-dependencies', "Don't show accepted dependencies") { |o| options.show_dependencies = false }
+  opt.on('--no-todos', "Don't show package todos") { |o| options.show_todos = false }
+  opt.on('--no-privacy', "Don't show privacy enforcement") { |o| options.show_privacy = false }
+  opt.on('--no-teams', "Don't show team colors") { |o| options.show_teams = false }
+
+  opt.on('--focus-on=PACKAGE', "Don't show privacy enforcement") { |o| options.focus_package = o }
+  opt.on('--only-edges-to-focus', "If focus is set, this shows only the edges to/from the focus node instead of all edges in the focussed graph. This only has effect when --focus-on is set.") { |o| options.show_only_edges_to_focus_package = true }
+
+  opt.on('--roll_nested_todos_into_top_level', "Don't show nested packages (not counting root). Connect edges to top-level package instead") { |o| options.roll_nested_todos_into_top_level = true }
+  opt.on('--focus_folder=FOLDER', "Draw package diagram only for packages in FOLDER") { |o| options.focus_folder = o }
+  opt.on('--no_nested_relationships', "Don't draw nested package relationships") { |o| options.show_nested_relationships = false }
+
+  opt.on('--remote-base-url=PACKAGE', "Link package nodes to an URL (affects graphviz SVG generation)") { |o| options.remote_base_url = o }
+
+  opt.on_tail("-h", "--help", "Show this message") do
+    puts opt
+    exit
+  end
+end.parse!
+
+puts VisualizePacks.package_graph!(
+  options, 
+  ParsePackwerk::Configuration.fetch.raw, 
+  Packs.all.map { |pack| ParsePackwerk.find(pack.name) }
```

Comment:
Stuff like this is why I wanted to merge `parse-packwerk` into `packs-specification`. I think it also makes a lot of sense conceptually (since at Gusto we think of packwerk as creating boundaries along a more fundamental idea of packs).